backend/src/backend/main.py

```diff
@@ -95,20 +98,41 @@ async def add_book(book_payload: BookAddSchema):
                 kind=KIND_TEXT,
             )
 
+    try:
+        title = await Title.objects.get_or_create(
+            ident=get_ident_from_name(book_payload.metadata["Name"])
+        )
+    except ValueError as exc:
+        raise HTTPException(status_code=400, detail=f"Exception: {exc}") from exc
```

Comment:
We caught that so we can provide better detail on the failure so inform the caller of the cause:

```py
f"Invalid Name metadata. Unable to build Title identifier: {exc}"
```